compiler/src/dmd/expression.d

```diff
@@ -5329,6 +5329,26 @@ extern (C++) final class CallExp : UnaExp
     }
 }
 
+/**
+ * Get the called function type from a call expression
+ * Params:
+ *   ce = function call expression. Must have had semantic analysis done.
+ * Returns: called function type, or `null` if error / no semantic analysis done
+ */
+TypeFunction calledFunctionType(CallExp ce)
```

Comment:
```suggestion
TypeFunction calleeType(CallExp ce)
```

`const`/`inout` would be good here although I expect its not possible.

Reply:
Indeed can't call `toBasetype` on an `inout` object

Reply:
The "called function" is the callee

Reply:
You don't know the caller but if the CallExp has been resolved properly you always know the callee